app/client/lib/ACUserManager.ts

```diff
@@ -0,0 +1,225 @@
+import {ACResults, ACIndex, ACItem, buildHighlightedDom, normalizeText} from "app/client/lib/ACIndex";
+import {cssSelectItem} from "app/client/lib/ACSelect";
+import {Autocomplete, IAutocompleteOptions} from "app/client/lib/autocomplete";
+import {cssMenuItem} from "popweasel";
+
+import {testId, colors, theme} from "app/client/ui2018/cssVars";
+import {menuCssClass} from "app/client/ui2018/menus";
+import {dom, DomElementArg, Holder, IDisposableOwner, Observable, styled, computed, Computed} from "grainjs";
+import {
+  cssEmailInput,
```

Comment:
Sorry to keep bugging you with unrelated things @LouisDelbosc, but would you be able to remove the `line-height: 42px` from this style? The input text looks off when it's selected, and I don't see any downsides with removing it. Thank you!